pkgs/tools/networking/mozwire/default.nix

```diff
@@ -8,12 +14,28 @@ rustPlatform.buildRustPackage rec {
     owner = "NilsIrl";
     repo = pname;
     rev = "v${version}";
-    sha256 = "01bj3c34x9ywxygsz4rdyw5gc9cz8x6zzl5fd7db8qy8bx2lhlr9";
+    hash = "sha256-KVNIRV/IY7Taaa7Q/01HnyX2Cvctk6+f79ynTgYbcgU=";
   };
 
+  cargoPatches = [
+    # The two patches below are unreleased and update clap to v3. The second one
+    # also accidentally updates socket2, which is needed in order to compile on
+    # Rust 1.64.
+    (fetchpatch {
+      name = "1-update-clap-and-fix-for-rust-1.64.patch";
+      url = "https://github.com/NilsIrl/MozWire/commit/ebf05d2667e92e85a3d94eb3eb08520b6e7c416b.patch";
+      hash = "sha256-CMSUlu4cskcxsy8PAipSev86o9nK/N3LyMRg8e9h6tc=";
+    })
+    (fetchpatch {
+      name = "2-update-clap-and-fix-for-rust-1.64.patch";
```

Comment:
```suggestion
      name = "update-clap-and-fix-for-rust-1.64-part-1.patch";
      url = "https://github.com/NilsIrl/MozWire/commit/ebf05d2667e92e85a3d94eb3eb08520b6e7c416b.patch";
      hash = "sha256-CMSUlu4cskcxsy8PAipSev86o9nK/N3LyMRg8e9h6tc=";
    })
    (fetchpatch {
      name = "update-clap-and-fix-for-rust-1.64-part-2.patch";
```

Reply:
The maintainer of `MozWire` created a new release so I dropped both of these. 🎉